packages/mdc-select/index.js

```diff
@@ -85,11 +65,10 @@ export class MDCSelect extends MDCComponent {
     if (bottomLineElement) {
       this.bottomLine_ = bottomLineFactory(bottomLineElement);
     }
-    this.selectedText_ = this.root_.querySelector(strings.SELECTED_TEXT_SELECTOR);
-    this.menuEl_ = this.root_.querySelector(strings.MENU_SELECTOR);
-    this.menu_ = menuFactory(this.menuEl_);
 
-    this.ripple = new MDCRipple(this.surface_);
+    if (this.root_.classList.contains(cssClasses.BOX)) {
+      this.ripple = new MDCRipple(this.root_);
```

Comment:
Only one of my two previous comments was resolved. Here's the other one:

We probably want to do something like what we do with checkbox/radio/text-field to forward the ripple component's event handlers to the select element. This should make activation work when pressing space, and should make focus styles work via ripple JS.

Specifically, I'm talking about e.g. this: https://github.com/material-components/material-components-web/blob/3f238212915cb781a1ca4cb4cad9da392a52bf22/packages/mdc-checkbox/index.js#L64-L65